volatility3/framework/symbols/windows/extensions/registry.py

```diff
@@ -249,6 +282,10 @@ def get_name(self) -> interfaces.objects.ObjectInterface:
         self.Name.count = namelength
         return self.Name.cast("string", max_length=namelength, encoding="latin-1")
 
+    def get_type(self) -> RegValueTypes:
```

Comment:
Additions like this should still bump the MINOR version number of the framework, but luckily nothing else will have to change (any plugins that call these methods though will have to depend on that later framework version).